internal/wire/frame_test.go

```diff
@@ -27,3 +27,43 @@ func TestProbingFrames(t *testing.T) {
 		require.Equal(t, expected, IsProbingFrame(f))
 	}
 }
+
+func TestIsProbingFrameType(t *testing.T) {
```

Comment:
This also looks too verbose, have a look at the tests in the ackhandler package for how to make this more compact.